bundles/org.openhab.binding.bmwconnecteddrive/src/main/java/org/openhab/binding/bmwconnecteddrive/internal/handler/ConnectedDriveProxy.java

```diff
@@ -290,7 +290,7 @@ private synchronized void updateToken() {
                 logger.debug("Authorization status {} reason {}", contentResponse.getStatus(),
                         contentResponse.getReason());
             }
-        } catch (InterruptedException | ExecutionException | TimeoutException e) {
+        } catch (InterruptedException | ExecutionException | TimeoutException | JsonSyntaxException e) {
```

Comment:
catching JsonSyntaxException here is redundant to the catch in line 283

Reply:
Reverted